tests/unit/test_model_card.py

```diff
@@ -1017,6 +1065,9 @@ def test_training_details_autodiscovery_from_model_overview(
     assert len(training_details.training_job_details.training_metrics) == len(
         SEARCH_TRAINING_JOB_EXAMPLE["Results"][0]["TrainingJob"]["FinalMetricDataList"]
     )
+    assert len(training_details.training_job_details.training_metrics) == len(
```

Comment:
`training_details.training_job_details.training_metrics` ? we should assert hyper_paramters

Reply:
Will change training_metrics to hyper_parameters.